minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-resources/src/main/resources/conf/bootstrap.conf

```diff
@@ -41,6 +42,11 @@ nifi.minifi.config=./conf/config.yml
 # How frequently the file specified by 'nifi.minifi.notifier.file.config.path' should be evaluated for changes.
 #nifi.minifi.notifier.file.polling.period.seconds=5
 
+# Rest change notifier configuration
+
+# Port on which the Jetty server will bind to, keep commented for a random open port
+#nifi.minifi.notifier.http.port=8338
+
```

Comment:
do you think it makes sense to include the stubbed out options for the mutual auth notifier configuration as well?